### DISCUSSION TITLE
Download this project!

### DISCUSSION THREAD

**Discussion by RedLeafSystems** [Dansoftowner/jSystemThemeDetector#29] [Q&A] [answered]
I don't know where and how to download and use this product! Can someone help me with this?
Thanks!

**Comment by vewert** [comment, marked as answer]
It is available from jitpack, and can be added as a dependency to your project using gradle, maven, etc.

See the sections called: **Using it with Gradle, Maven... etc**, the README, for more information. Especially note that you need to add jitpack as an available maven repository using:

```
repositories {
	...
	maven { url 'https://jitpack.io' }
}
```
